arduino/cores/status.go

```diff
@@ -44,13 +40,13 @@ type Package struct {
 	Email      string               // Email of maintainer.
 	Platforms  map[string]*Platform // The platforms in the system.
 	Tools      map[string]*Tool     // The tools in the system.
-	Packages   *Packages            `json:"-"`
```

Comment:
now that `Packages` is aliased to a map, no need to send around a pointer, map is already a reference